public/js/0.js

```diff
@@ -0,0 +1,1703 @@
+(window["webpackJsonp"] = window["webpackJsonp"] || []).push([[0],{
```

Comment:
This is the axios chunk, gets loaded on every page that needs axios. Won't get loaded if no axios is imported in the component.